### ISSUE TITLE
Codekit and Zurb Foundation (4.3.1)

### ISSUE THREAD

**Issue by replete** [bdkjones/CodeKit-1-Old#669]
**tl;dr** I fixed this by using external `compass` and `sass` executables, but I have not previously had to do this.

_My setup:_
- Brand new Macbook Air 13" 2013 (urgently purchased on-site as my MBA11" died :-1: )
- Mac OS X 10.8.4
- Latest Codekit (1.81 - 8326)

``` bash
$ sudo gem list

*** LOCAL GEMS ***

chunky_png (1.2.8)
compass (0.12.2)
fssm (0.2.10)
rubygems-update (2.1.1)
sass (3.2.10)
zurb-foundation (4.3.1)
```

I'm really stuck with this codeKit error, in all of my projects. I've used Codekit and compass for over a year while contracting, so I'm really familiar with it (at least, I thought I was):

```
Compass was unable to compile one or more files in the project: 

LoadError on line 161 of /Applications/CodeKit.app/Contents/Resources/engines/compass/bin/../lib/compass/configuration/data.rb: no such file to load -- zurb-foundation
Run with --trace to see the full backtrace

(This action was triggered by a change to _typography.scss)
```

For now, I'm using `compass watch` to at least get sass working, but this isn't ideal.

_Updated_: Fixed by using external `compass` and `sass` executables in Codekit (thought they seem a little slower :/).

**Comment by bdkjones** [comment]
You've always needed to switch CodeKit to an external Compass compiler to use Zurb directly. The only time you don't need to do this is if you download Zurb as stand-alone stylesheets and simply copy those into your project folder. Cheers!
